### PR TITLE
Allow other users to do certain actions as the PR opener.

### PR DESCRIPTION
**Pull request by ellisonmarks**

### CODE REVIEW

File: .github/workflows/tacos_unlock.yml [hunk 1]
```diff
@@ -90,16 +90,20 @@ jobs:
...
+                  false
+                  || github.event.action == 'closed'
+                  || github.event.action == 'converted_to_draft'
+              )
 
               && github.event.pull_request.user.login
 
-              || github.actor
+              || github.triggering_actor
             }}
```
Comment (bukzor) [inline .github/workflows/tacos_unlock.yml:107]: Do parens make the implicit ternary any clearer? Not sure.

```suggestion
            ${{
              (
                true
                && github.event_name == 'pull_request'
                && (
                  false
                  || github.event.action == 'closed'
                  || github.event.action == 'converted_to_draft'
                )
                && github.event.pull_request.user.login
              ) || github.triggering_actor
            }}
```

Comment (ellisonmarks) [reply .github/workflows/tacos_unlock.yml:107]: Not particularly more than the whitespace, in my eyes.

File: .github/workflows/tacos_plan.yml [hunk 2]
```diff
@@ -91,6 +91,23 @@ jobs:
...
         with:
           ssh-private-key: ${{ secrets.ssh-private-key }}
+          # Use author's username for the following actions
+          user: |
+            ${{
+              true
+              && github.event_name == 'pull_request'
+              && (
+                  false
+                  || github.event.action == 'labeled'
```
Comment (bukzor) [inline .github/workflows/tacos_plan.yml:101]: should be && the actual label

File: .github/workflows/tacos_plan.yml [hunk 3]
```diff
@@ -91,6 +91,23 @@ jobs:
         uses: ./tacos-gha/.github/actions/setup
         with:
           ssh-private-key: ${{ secrets.ssh-private-key }}
+          # Use author's username for the following actions
```
Comment (bukzor) [inline .github/workflows/tacos_plan.yml:94]: Please leave a reference to our doc

File: .github/workflows/tacos_unlock.yml [hunk 4]
```diff
@@ -90,16 +90,20 @@ jobs:
         uses: ./tacos-gha/.github/actions/setup
         with:
           ssh-private-key: ${{ secrets.ssh-private-key }}
-          # Use author's username when unlocking for closed PRs
+          # Use author's username for the following actions
```
Comment (bukzor) [inline .github/workflows/tacos_unlock.yml:93]: # We explicitly list the low-concern actions, during which users will use
# the identity of the original PR author.

File: .github/workflows/tacos_plan.yml (hunk 2 above)
Comment (ellisonmarks) [reply .github/workflows/tacos_plan.yml]: added the condition.

**Review by bukzor — APPROVED** [review]

**Pull request merged**